pyntc/devices/eos_device.py

```diff
@@ -46,6 +47,18 @@ def __init__(self, host, username, password, transport="http", timeout=60, **kwa
         self.timeout = timeout
         self.connection = eos_connect(transport, host=host, username=username, password=password, timeout=timeout)
         self.native = EOSNative(self.connection)
+        # _connected indicates Netmiko ssh connection
+        self._connected = False
+        # ftp corresponds to file transfer and openning a netmiko connection
+        if self.transport == "ftp":
```

Comment:
I don't think we would expect transport to be ftp. They will have an http/https transport, but if they use the file copy method, then we would establish an scp connection.

Reply:
It was an unfortunate name. I wanted to distinguish between file copy and all other functions so that we do not create a netmiko ssh connection when we do not need it. Changed to address comment.

Reply:
Changes made per our conversation